packages/lib/contracts/Initializable.sol

```diff
@@ -32,12 +32,18 @@ contract Initializable {
     require(initializing || isConstructor() || !initialized, "Contract instance has already been initialized");
 
     bool wasInitializing = initializing;
-    initializing = true;
-    initialized = true;
+    if (!wasInitializing) {
```

Comment:
I personally think that `if (initializing != true)` would result in more self-evident code. What do you guys think?